be/referral.go

```diff
@@ -0,0 +1,116 @@
+package be
+
+import (
+	"cleanapp/common"
+	"net/http"
+
+	"github.com/apex/log"
+	"github.com/gin-gonic/gin"
+)
+
+type ReferralQuery struct {
```

Comment:
Would this be a good place to documents what these fields are?
Normally this is self-evident, but how one could guess that RefKey = "IP4:Width:Length"?
Or even tougher, that RefValue = ID of the user who referred?